DQM/SiPixelPhase1Heterogeneous/plugins/SiPixelPhase1CompareTrackSoA.cc

```diff
@@ -64,6 +64,7 @@ namespace {
 
 class SiPixelPhase1CompareTrackSoA : public DQMEDAnalyzer {
 public:
+  using PixelTrackHeterogeneous = PixelTrackHeterogeneousT<pixelTopology::Phase1>;
```

Comment:
I think it would be clearer (albeit more intrusive, requiring changes in the rest of the code) if this had "Phase1" in the name.

Alternatively, the whole module could be templated on the Phase1/Phase2 trait, in order to have the validation also for Phase 2 ?
This could be a task for a further PR, of course.